Pc #125

ferruirob opened the pull request:
Añadida las correcciones a las incidencias detectadas:
-Añadido cambio al modelo
-Reconfigurado el método para generar fecha de cancelación, ahora se obtiene la fecha de fin de periodo de stripe
-Creado método para que desde el perfil, puedas traerte la fecha de cancelación desde stripe y se pueda usar

ferruirob committed:
Anadido tests de la tienda PC-#122

Creado test para comprobar si se accede logeado o no y cancelar suscripcion

ferruirob committed:
Anadido correciones a las incidencias

Modificado el modelo de premium, cambiado metodo de obtencion de fecha de cancelacion ahora se trae de stripe la fecha
de final de periodo y creado metodo para perfil de obtencion de fecha de cancelacion

jorandmol reviewed (changes requested):
Funciona todo bien, aunque haría algunas modificaciones:
- Eliminar el archivo charge.html ya que no es necesario.
- Cambiar las URLs al español.
- Asegurar el control de las excepciones en el controlador. Se realiza fácilmente envolviendo todo en un bloque try/catch que redirija a la vista base/error.html
- Modificar el método de obtención de fecha de cancelación para que pueda ser consultado desde la vista (se puede usar como ejemplo la función "ValorarPublicacionView" del controlador ranking/views.py, en el que se devuelve un json).

javsolgar committed:
Se han añadido algunas propuestas-#125

Se han cambiado las urls, se han añadido el control de excepciones y se ha eliminado el charge.html

javsolgar committed:
Se han completado los cambios propuestos-#125

Se ha implementado la respuesta de la fecha de cancelación como un JSON

javsolgar commented:
Se han añadido los cambios que se han sugerido

jorandmol reviewed (approved):
He comprobado satisfactoriamente que los cambios propuestos se han realizado y que funcionan como se espera
